gradle/scripts/README.md

```diff
@@ -510,6 +510,19 @@ generate Maven BOM based on the dependencies specified in `dependencies.toml`.
 `bom` flag implies `publish` flag, which means the BOM will be uploaded to a
 Maven repository by `./gradlew publish`.
 
+## Sharing [dependency versions](https://docs.gradle.org/current/userguide/platforms.html#sec:version-catalog-plugin) with `version-catalog` flag
+
+If you configure a project with `version-catalog` flag, the project will be configured to
```

Comment:
The gradle script seems to be using `version-catalog[s]` whereas the documentation seems to be describing the flag as `version-catalog`.

Ditto for the rest of the documentation
```suggestion
If you configure a project with `version-catalogs` flag, the project will be configured to
```